src/main/webapp/app/shared/metis/postings-create-edit-modal/post-create-edit-modal/post-create-edit-modal.component.ts

```diff
@@ -13,14 +13,18 @@ const TITLE_MAX_LENGTH = 200;
     templateUrl: './post-create-edit-modal.component.html',
 })
 export class PostCreateEditModalComponent extends PostingsCreateEditModalDirective<Post> {
+    tags: string[];
+
     constructor(protected metisService: MetisService, protected modalService: NgbModal, protected formBuilder: FormBuilder) {
         super(metisService, modalService, formBuilder);
+        this.tags = this.posting?.tags ?? [];
```

Comment:
Any opinion on putting that into the constructor and not in an ngOnInit? I believe that we hardly ever assign attributes in the constructor